src/gasPriceOracle/oracle.e2e.ts

```diff
@@ -13,19 +13,22 @@ const dummyLogger = winston.createLogger({
 type FeeData = providers.FeeData;
 
 class MockedProvider extends providers.JsonRpcProvider {
-  public feeData: BigNumber | number | string | undefined;
-  public gasPrice: BigNumber | number | string | undefined;
+  // Unknown type => exercise our validation logic
+  public feeData: unknown;
```

Comment:
Can we use different names like `unknownFeeData` to make it clear that these are purposefully unknown type?

Reply:
I merged this before I saw your comment, so have published another PR: https://github.com/across-protocol/sdk-v2/pull/119